### PR TITLE
Added Group `repo`

### PR DESCRIPTION
**Pull request by RichardLitt**
Also straightforward.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1696,8 +1696,53 @@ root will not be listable, as it is virtual. Access known paths directly.
...
+
+    + Attributes (ndjson)
+        - Key (Multihash)
+
+
+    + Body
+
+        ```
+        {"Key":"QmNLvkCDV6ZjUJsEwGNporYBuZdhWT6q7TBVYQwwRv12HT"}
+        {"Key":"QmNMkhF7m6CjJiNLsDbGrhfTbpjfTUu6qG3SVSMQ32isWb"}
```
Comment (dignifiedquire) [inline apiary.apib]: What are these, the objects that were removed?

Comment (RichardLitt) [inline apiary.apib]: cc @whyrusleeping

**Comment by RichardLitt** [comment]
Add a note saying that the keys are what has been removed.

File: apiary.apib (hunk 1 above)
Comment (diasdavid) [inline apiary.apib]: ping

**Comment by daviddias** [comment]
Anything missing for the merge?

**Comment by daviddias** [comment]
This is missing `repo stat`

**Comment by RichardLitt** [comment]
Needs:
- [ ] Add note about keys
- [ ] Add in `repo stat`
- [ ] Check all commands
- [ ] Update headers

**Comment by RichardLitt** [comment]
Opened. https://github.com/ipfs/go-ipfs/pull/2585

**Pull request merged**